libzecale/core/nested_transaction.hpp

```diff
@@ -19,25 +19,29 @@ template<typename nppT, typename nsnarkT> class nested_transaction
 private:
     std::string _application_name;
     std::shared_ptr<libzeth::extended_proof<nppT, nsnarkT>> _extended_proof;
+    std::vector<uint8_t> _parameters;
```

Comment:
What prevents malicious manipulation of these `_parameters` by the aggregator? i.e. what proves that `nested_parameters` in the `AggregatedTransaction` proto message is honestly computed as the concatenation of all `parameters` from the popped nested transactions processed as part of the last batch?